### DISCUSSION TITLE
How can I stop qalc from changing units of values less than 1?

### DISCUSSION THREAD

**Discussion by SayYesToTrees** [Qalculate/libqalculate#519] [Q&A] [answered]
For my purposes it would be very handy if I could make for instance 0.3 m not be stored in a variable as millimeters. I have tried changing the auto conversion to off but to no avail. In most applications I see why this is the functionality but for use in my class it just makes me have to double check that the units didn't change and I have to perform a 'to' operation to get the correct units and I would prefer not to have to do that.

**Comment by hanna-kn** [comment, marked as answer]
To disable automatic use of unit prefixes for display of values use the `prefixes` option, e.g. set` prefixes off`, or shorter `set pref 0`, or as a command line switch `-s "pref 0"`.

Prefixes are however never stored in variables, but unfortunately always used for display of variable values when using the `list` and `info` commands. I will change this so that the current status of the `prefixes` option will be used. This only applies to variables saved during the current session, not persistent variables after qalc has been restarted.

**Reply by SayYesToTrees** [reply]
Thank you so much!!! I knew I was just missing the option.
